.github/workflows/ci-pipeline.yaml

```diff
@@ -36,3 +36,38 @@ jobs:
         SLACK_MESSAGE: 'Build and test workflow running'
         SLACK_TITLE: 'Build and Test Notification'
 
+
+  docker:
+    needs: build
+    #if: github.repository == 'mpursell/DevOps-Course-Starter/module8'
+    runs-on: ubuntu-latest
+    environment: Build and Test environment
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    
+    # Setup docker buildx
+    # https://github.com/docker/setup-qemu-action
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
```

Comment:
You don't actually need QEMU and buildx, e.g. you're not emulating another architecture or anything. But no harm in including these steps